docs/data/data-grid/row-dimensions/row-dimensions.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Data Grid - Row dimensions
+---
+
+# Data grid - Row dimensions
+
+<p class="description">Customize the dimensions of your rows.</p>
+
+## Static row height
+
+By default, the rows have a height of 52 pixels.
```

Comment:
@samuelsycamore I feel like we could be more explicit here about the difference between rowHeight and density.